### ISSUE TITLE
Allow more than one config file

### ISSUE THREAD

**Issue by gmcinnes** [papertrail/remote_syslog2#63]
It might be handy if you could allow remote_syslog2 to follow the directory convention of:

````
- - etc
  | - remote_syslog.d
    |- application1_files.yml
     - application2_files.yml
````

Each yml file under there would potentially allow for the inclusion or exclusion of its own files.  That way, if I say, deploy a web app to a box, I can have it put it's own .yml file in there, and ask remote_syslog2 to reload its config.

By doing that, the web app would be able to manage its own concerns without having to know about global things, such as the logging port for papertrail, or whether to use tls or not.

What do you think?

**Comment by troy** [comment]
Good idea. I can definitely see value to it. Any interest in implementing it?

**Comment by gmcinnes** [comment]
I could give it a go, but it will be a *long* time before I get to it, because I haven't played with Go yet.

-g

**Comment by troy** [comment]
Cool, thanks @gmcinnes. I'll leave this open; if anyone else wants to jump on it, feel free, and if not, we'll prioritize alongside other improvements.
